app/models/concerns/contest_attributes.rb

```diff
@@ -27,10 +27,10 @@ def currently_in_contest?
     started? && !ended?
   end
 
-  def max_score(sp_count = short_problems.count, lp_count = long_problems.count)
+  def max_score(sp_count = short_problems.count, lp_count = long_problems.count, id_contest)
```

Comment:
Why does this function take in the `id_contest`?

This concern is loaded in the `Contest` model. It will be a method in that model. You can access the id of that contest directly with the `id` attribute.